### DISCUSSION TITLE
How to make a round highlighting on touch instead of square?

### DISCUSSION THREAD

**Discussion by rvalitov** [taiga-family/taiga-ui#849] [Q&A] [answered]
On touch devices when you touch an element, it's being highlighted. In native apps if we click a round button, then it's highlighted according to its geometry, which is also round. But in Taiga all elements are highlighted square, no matter of their original geometry.

For example, I want to use MarkerIcon as a circle button. Sample code:
```html
<a
          *ngIf="canGoBack"
          (click)="onClickBack()"
        >
          <tui-marker-icon
            class="tui-space_right-3"
            src="tuiIconArrowLeftLarge"
            mode="link"
          ></tui-marker-icon>
</a>
```

In this case the button looks like follow:
![image](https://user-images.githubusercontent.com/16267156/136740383-2153a57d-b8b0-4c94-8872-9b04bb3ec5a9.png)

When I touch it, the highlighting is square:
![image](https://user-images.githubusercontent.com/16267156/136740474-a15ce989-311f-456b-b211-3825cd6e8ba8.png)

I want it to be round for round elements. Any ideas how I can achieve that?

**Comment by waterplea** [comment, marked as answer]
The best thing you can do is remove this highlight with `-webkit-tap-highlight-color: transparent;` which is done on Taiga UI components. But in your case this highlight is on the `a` tag. Perhaps we should add `a[tuiMarkerIcon]` to selectors so icon can be put directly on a link.
